src/common/components/SupportSidebar/MentoringConsumables.jsx

```diff
@@ -144,6 +144,18 @@ function MentoringConsumables({
     }
   }, [allMentorsAvailable]);
 
+  const checkForConsumableAvailable = (consumableList, serviceSelected) => {
+    const filteredConsumables = consumableList.filter((consumable) => consumable?.mentorship_services?.some((service) => serviceSelected?.slug === service.slug));
```

Comment:
I think this doesn't work because the consumable includes the slug of the mentorship service set, then you should do a request to get the mentoring sessions of the mentorship service set and compare them with the current of the widget